requirements/base.txt

```diff
@@ -27,19 +27,21 @@ pbr==5.5.1
     # via stevedore
 psutil==5.8.0
     # via edx-django-utils
-pyjwt==2.0.0
+pyjwt==1.7.1
```

Comment:
A bunch of packages were downgraded here; the job is probably still configured to run on Python 3.5, could you fix that and re-run it?